static/js/publisher/release/components/notification.js

```diff
@@ -43,13 +45,25 @@ export default class Notification extends Component {
 }
 
 Notification.propTypes = {
-  children: PropTypes.node,
+  content: PropTypes.node,
```

Comment:
These are breaking changes.
This component is used in other places (like RevisionsList component) when it uses `children`.
Currently it doesn't display at all and throws errors in console.

To see it you need to look at snap with some devmode revisions in revisions history. Notification should appear warning about devmode revisions.

Reply:
I think we need 2 components:
1st that is previous implementation of purely visual notification - so any component can render it anywhere.
2nd is "global" notification connected to redux store that takes content from the store and is shown/hidden by actions on the store.